examples/basics/thermal_basics.py

```diff
@@ -9,123 +9,173 @@
+mc.set_variable("Motor_Type", 0)
+print("Initialisation completed.")
 
 # %%
-# Set the heat transfer correlation.
-mcad.set_variable("Calc/Input_h[WJ]_Rear_Housing", 1)
-mcad.set_array_variable("HousingWJ_CalcInputH_A", 0, 1)
-
-wj_fluid_k = mcad.get_variable("WJ_Fluid_Thermal_Conductivity")
-wj_fluid_rho = mcad.get_variable("WJ_Fluid_Density")
-wj_fluid_mu = mcad.get_variable("WJ_Fluid_Dynamic_Viscosity")
-wj_fluid_u_a = mcad.get_array_variable("HousingWJ_Velocity_A", 0)
-wj_fluid_u_r = mcad.get_variable("WJ_Channel_Fluid_Velocity_[Rear]")
+# Set up analysis
+# ---------------
+# Setting up the analysis consists of setting the geometry and thermal input data parameters and
+# saving the file.
+#
+# Geometry setup
+# ~~~~~~~~~~~~~~
+# Display the **Scripting** tab and set the housing outer diameter to 250 mm.
+#
+# .. note::
+#    To set parameter values via automation with the GUI visible, a different tab must be displayed.
+#    The **Scripting** tab is chosen because no parameters on this tab will be changed in this
+#    script. It is best practice to display the **Scripting** tab when changing input parameters by
+#    automation. This is required only if the GUI is shown.
+mc.display_screen("Scripting")
+mc.set_variable("Housing_Dia", 250)
+
+# %%
+# Thermal input data setup
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Set the housing water jacket cooling parameters:
+#
+# * Fluid volume flow rate (in units of m\ :sup:`3`/s).
+#
+# * Fluid inlet temperature (in units of °C).
+#
+# * Fluid used by the HWJ (from material database).
+mc.set_variable("WJ_Fluid_Volume_Flow_Rate", 0.002)
+mc.set_variable("WJ_Fluid_Inlet_Temperature", 25)
+mc.set_fluid("HousingWJFluid", "Dynalene HF-LO")
+
+# %%
+# Disable the option for *Active Cooling Only*, so that the HWJ channel also cools the front and
+# rear housing segments.
+mc.set_variable("Water_Jacket_Active_Cooling_Only", 0)
+
+# %%
+# Enable the *Input h* option to allow user input of the heat transfer coefficients for HWJ cooling
+# of the rear and active segments of the housing.
+mc.set_variable("Calc/Input_h[WJ]_Rear_Housing", 1)
+mc.set_array_variable("HousingWJ_CalcInputH_A", 0, 1)
+
+# %%
+# Get fluid properties (thermal conductivity, density and dynamic viscosity) and the fluid
+# velocities for the rear and active segments of the housing. Calculate new heat transfer
+# coefficients for the rear and active segments of the housing. Run a steady state thermal
+# calculation to ensure that the fluid properties and velocities are correctly calculated based on
+# the model changes made earlier.
+#
+# .. note::
+#    Some Motor-CAD output parameters are not automatically updated when an input parameter is set.
+#    Outputs will be updated when a steady state thermal calculation runs. Alternatively, the
+#    relevant tab (for example, *Input Data -> Housing Water Jacket -> Heat Transfer*) can be
+#    displayed - however this may not be suitable when running Motor-CAD without the GUI.
```

Comment:
```suggestion
#    Outputs are updated when a steady state thermal calculation runs. Alternatively, the
#    relevant tab (for example, **Input Data -> Housing Water Jacket -> Heat Transfer**) can be
#    shown. However, this might not be suitable when running Motor-CAD without the GUI.
```